File: Listas de Exercícios (Python)/Python/Lista 1 - Casa/1.py
print("Conversor: Real -> Dólar\n")

print("--Valores decimais devem ser digitados com (.)!--")

R=float(input("Valor (R$): "))

D=R/2.40; D="{:.2f}".format(D)

print(f"\nValor: ${D}\n")

File: Listas de Exercícios (C)/C/Lista 1 - Casa/1.c
#include <stdio.h>

int main()
{
    float R, D;
    
    printf("Conversor: Real -> Dólar\n\n");
    
    printf("--Valores decimais devem ser digitados com (.)!--\n");

    printf("Valor (R$): "); scanf("%f", &R);
    
    D=R/2.40;
    
    printf("\nValor: $%.2f\n", D);
}